rllib/algorithms/cql/torch/cql_torch_learner.py

```diff
@@ -230,11 +230,9 @@ def compute_loss_for_module(
         # TODO (simon): Add loss keys for langrangian, if needed.
         # TODO (simon): Add only here then the Langrange parameter optimization.
         if config.twin_q:
-            self.metrics.log_dict(
```

Comment:
Was this another `log_dict` logging issue?

Reply:
I don't think it's this important anymore whether we use `log_dict` once vs `log_values` n times (n = number of key/value pairs to log).
But we'll keep an eye on CPU flamegraphs throughout the near future to make sure, everything is super optimized.